client/fs_endpoint_test.go

```diff
@@ -597,11 +601,14 @@ OUTER:
 
 type ReadWriteCloseChecker struct {
 	io.ReadWriteCloser
+	l      sync.Mutex
 	Closed bool
 }
 
 func (r *ReadWriteCloseChecker) Close() error {
+	r.l.Lock()
 	r.Closed = true
+	r.l.Lock()
```

Comment:
`r.l.Unlock()`?

Reply:
oops - i thought i fixed this :( - i'll repush.